tsconfig.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This is _awesome_.

Reply:
I pulled this down locally and the first compile was fast, and the second with some changes was almost instant.